OCCommunicationLib/OCCommunicationLib/OCCommunication.h

```diff
@@ -562,12 +562,12 @@ typedef enum {
  * @param path -> NSString server path
  * @param sharedOCCommunication -> OCCommunication Singleton of communication to add the operation on the queue.
  *
- * @return BOOL in the success about the support
+ * @return BOOL in the success about the support of Share and Sharee APIs
  *
  */
-- (void) hasServerShareSupport:(NSString*) path onCommunication:(OCCommunication *)sharedOCCommunication
-                successRequest:(void(^)(NSHTTPURLResponse *response, BOOL hasSupport, NSString *redirectedServer)) success
-                failureRequest:(void(^)(NSHTTPURLResponse *response, NSError *error)) failure;
+- (void) hasServerShareAndShareeSupport:(NSString*) path onCommunication:(OCCommunication *)sharedOCCommunication
+                         successRequest:(void(^)(NSHTTPURLResponse *response, BOOL hasShareSupport, BOOL hasShareeSupport, NSString *redirectedServer)) success
```

Comment:
BOOL hasShareSupport, BOOL hasShareeSupport

Two variables with similar name... maybe it is the right way but at least in the comment of the method should appear a description about what is the meaning and the difference on all the parameter os this method